nntrainer/layers/layer_node.h

```diff
@@ -167,6 +166,23 @@ class LayerNode final : public ml::train::Layer, public GraphNode {
    */
   void setInputConnectionName(unsigned nth, const std::string &name);
 
+  /**
+   * @brief Set the Output Connection
+   * @note Each output must be identified only ONCE.
+   * @note when nth comes, getNumOutput() expends to nth + 1 as resize occurs.
+   * Please also notice none identified intermediate output (or mismatch between
+   * actual number of out tensors and output) is allowed but will produce
+   * warning, this implies that the output is not used else where.
```

Comment:
how about an error instead of warning?

Reply:
Was thinking about this but I think there will be usage for this.

pytorch equivalent for
eg) out, _ = self.lstm(input)